architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -296,15 +289,24 @@ for the tag "2.0":
   }
 }
 ----
+
+You can also do this using the `oc tag` command:
+
+`oc tag ruby:latest ruby:2.0`
+
 ====
 
 === Image Stream Mappings
-When the integrated OpenShift docker registry receives a new image, a
-notification is sent to OpenShift via the `*ImageStreamMapping*` route. The
-registry informs OpenShift of the image's namespace, name, tag, and docker
-metadata. OpenShift uses this information to create a new image and to update
-the appropriate image stream with the mapping between the image's tag and the
-image itself.
+When the integrated OpenShift Docker Registry receives a new image, it creates
```

Comment:
I notice you usually initcap "Docker Registry". Preference, or based on another reason?